CHANGELOG.asciidoc

```diff
@@ -2,6 +2,153 @@
 // each other on Github, so don't worry too much on using the right prefix.
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
+
+[[release-notes-8.4.2]]
+=== Beats version 8.4.2
+https://github.com/elastic/beats/compare/v8.4.1\...v8.4.2[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+
+*Auditbeat*
+
+
+*Filebeat*
+- [Httpjson]- Fix for pagination at root level not working when used with chaining.
+
+*Heartbeat*
+
+
+*Metricbeat*
+
+
+*Packetbeat*
+
+
+*Winlogbeat*
+
+
+*Functionbeat*
+
+
+==== Bugfixes
+
+*Affecting all Beats*
+- Fix in AWS related services initialisation relying on custom endpoint resolver. {issue}32888[32888] {pull}32921[32921]
+
+*Auditbeat*
+
+
+*Filebeat*
+
+
+*Heartbeat*
+
+
+*Metricbeat*
+
+- Fix and improve AWS metric period calculation to avoid zero-length intervals {pull}32724[32724]
+- Add missing cluster metadata to k8s module metricsets {pull}32979[32979]
+- Add missing cluster metadata to k8s module metricsets {pull}32979[32979] {pull}33032[33032]
```

Comment:
```suggestion
- Add missing cluster metadata to k8s module metricsets {pull}32979[32979] {pull}33032[33032]
```

I think we only need the second one, this looks duplicated (probably from merge conflict resolution).